README.md

```diff
@@ -883,14 +883,14 @@ $ gem install google-cloud-video_intelligence
 
 ## Supported Ruby Versions
 
-These libraries are currently supported on Ruby 2.0+.
-
-However, Ruby 2.3 or later is strongly recommended, as earlier releases have
-reached or are nearing end-of-life. After June 1, 2018, Google will provide
-official support only for Ruby versions that are considered current and
-supported by Ruby Core (that is, Ruby versions that are either in normal
-maintenance or in security maintenance).
-See https://www.ruby-lang.org/en/downloads/branches/ for further details.
+These libraries are currently supported on Ruby 2.3+.
+
+Google provides official support for Ruby versions that are actively supported
+by Ruby Core (that is, Ruby versions that are either in normal maintenance or
+in security maintenance, and not end of life). Currently, this means Ruby 2.3
```

Comment:
I would suggest changing the capitalization and punctuation to make a new sentence within the parentheses. Same for Stackdriver README, below.

> by Ruby Core. (That is, Ruby versions that are either in normal maintenance or
+in security maintenance, and not end of life.)

Reply:
What's in the parentheses is not a complete sentence. I could change it to an em-dash.